packages/platform-server/src/utils.ts

```diff
@@ -60,6 +73,7 @@ async function _render(platformRef: PlatformRef, applicationRef: ApplicationRef)
 
   const platformState = platformRef.injector.get(PlatformState);
   if (applicationRef.injector.get(IS_HYDRATION_DOM_REUSE_ENABLED, false)) {
+    appendSsrContentIntegrityMarker(platformState.getDocument());
     annotateForHydration(applicationRef, platformState.getDocument());
```

Comment:
Minor comment, just to avoid calling the method twice:
```suggestion
  if (applicationRef.injector.get(IS_HYDRATION_DOM_REUSE_ENABLED, false)) {
    const doc = platformState.getDocument();
    appendSsrContentIntegrityMarker(doc);
    annotateForHydration(applicationRef, doc);
```